sc/src/test/java/uk/ac/ebi/atlas/tsne/TSnePointTest.java

```diff
@@ -15,6 +15,9 @@ public void create() throws Exception {
 
     @Test
     public void compareTo() throws Exception {
-        assertThat(TSnePoint.create(0.0, 0.0, 0.0, "B").compareTo(TSnePoint.create(1.0, 1.0, 1.0, "A")), is(greaterThan(0)));
+        assertThat(
```

Comment:
This relies on people remembering how `compareTo` works and what it means when it returns a -1/0/1 . Possible improvement: make a list, sort it, and assert what the outcome is?

You could use this opportunity to explain the future developers why you want the TSnePoints compared by name.